packages/unit-jest/src/install.js

```diff
@@ -68,6 +68,14 @@ module.exports = function (api) {
     }typescript`,
   );
 
+  if (api.prompts.options.includes('swc')) {
+    api.render(`./templates/swc`)
```

Comment:
I guess this relies on the fact that ts-jest files are scaffolded first, so we're overwriting the scaffolded ones here
It's ok to rely on this kind of execution-order assumptions due to it being experimental, but I'd make it explicit with a comment